.github/workflows/cypress.yaml

```diff
@@ -84,7 +80,7 @@ jobs:
                       core.setOutput("email", response.data.author.email);
 
             # Only run Percy when it is demanded or we are running the daily build
-            - name: Enable Percy if X-Needs-Percy
+            - name: Enable Percy
```

Comment:
incidentally: this "if X-Needs-Percy" was just a lie. Removed it as I don't think there is any particular benefit to repeating the exact conditions under which the step will run: as demonstrated, it is just a thing that we fail to maintain.